run.sh

```diff
@@ -9,21 +9,44 @@ TARGET=${TARGET:-armv7-unknown-linux-gnueabihf}
 TARGET_SSH=${TARGET_SSH:-pi@raspberrypi.local}
 PROFILE=${PROFILE:-debug}
 RUN=${RUN:-1}
+USE_SYSTEMD=${USE_SYSTEMD:-1}
 
-if [ $PROFILE = release ]
-then
-    time cross build --target $TARGET --release
-elif [ $PROFILE = debug ]
-then
-    time cross build --target $TARGET
+if [ $PROFILE = release ]; then
+    PROFILE_FLAG=--release
+elif [ $PROFILE = debug ]; then
+    PROFILE_FLAG=''
 else
     echo "Invalid profile '$PROFILE'"
     exit 1
 fi
 
+time cross build $PROFILE_FLAG --target $TARGET --bin publish-mqtt
+
 time rsync --progress target/$TARGET/$PROFILE/publish-mqtt $TARGET_SSH:publish-mqtt
 
-if [ $RUN -eq 1 ]
-then
-    ssh $TARGET_SSH ./publish-mqtt
+SERVICE_FILE="
+[Unit]
+Description=publish-mqtt - send temperature readings to mqtt
+After=network.target
+
+[Service]
+Type=simple
+User=pi
+WorkingDirectory=/home/pi
+Environment=RUST_BACKTRACE=1
+ExecStart=/home/pi/publish-mqtt
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+"
```

Comment:
This seems like it should be in a separate file, which we could also use at some point in the future to build release packages.